docs/monthday.md

```diff
@@ -239,6 +239,12 @@ function reviver(key, value) {
 JSON.parse(str, reviver);
 ```
 
+### monthDay.**valueOf**()
+
+This method overrides `Object.prototype.valueOf()` and always throws an exception.
+This is because it's not possible to compare `Temporal.MonthDay` objects with the relational operators `<`, `<=`, `>`, or `>=`.
+Instead, use `absolute.equals()` to check for equality.
```

Comment:
Is this correct? In this case, should we include a simple example for converting a `MonthDay` to an `Absolute`? That said, I believe it'd be quite alright to convert just to a `Date`.

Reply:
Not correct, it was a copypasta error :smile: